recipes/botan/all/conanfile.py

```diff
@@ -64,7 +65,8 @@ class BotanConan(ConanFile):
                        'with_armv8crypto': True,
                        'with_powercrypto': True,
                        'enable_modules': None,
-                       'system_cert_bundle': None}
+                       'system_cert_bundle': None,
+                       'module_policy': None}
```

Comment:
What is the default policy?

Reply:
Per default no policy is used. The user can use include or remove whatever modules he want.

Reply:
Thanks for the reply.